src/aoclient.cpp

```diff
@@ -216,7 +216,26 @@ void AOClient::setHwid(QString p_hwid)
 
 void AOClient::sendServerMessage(QString message)
 {
-    sendPacket("CT", {"Server", message, "1"});
+    QSettings config("config/config.ini", QSettings::IniFormat);
+    config.beginGroup("Options");
+    QString hostname = config.value("hostname", "Server").toString();
+    sendPacket("CT", {hostname, message, "1"});
+}
+
+void AOClient::sendServerMessageArea(QString message)
+{
+    QSettings config("config/config.ini", QSettings::IniFormat);
+    config.beginGroup("Options");
+    QString hostname = config.value("hostname", "Server").toString();
```

Comment:
this is the same code as above and it looks ugly

can you write a method in the server class that gets the hostname for us, or better, one that sends system OOC messages to the specified audience